app/assets/javascripts/discourse/app/templates/components/parent-category-row.hbs

```diff
@@ -5,7 +5,7 @@
       {{category-title-link category=category}}
       {{#if category.description_excerpt}}
         <div class="category-description">
-          {{dir-span category.description_excerpt}}
+          {{dir-span category.description_excerpt false}}
```

Comment:
Sorry revisit an already approved and merged PR, but I dislike having unnamed boolean parameters. It's really not clear to me from the call site what this parameter is doing. Could you change it to be `escape="false"` or perhaps even  `htmlSafe="true"`

Reply:
Yep, I'll do that. I like that much better as well. Thanks for the input :)